lib/elasticsearch_models/query/response.rb

```diff
@@ -10,10 +10,11 @@ def initialize(message, original_exception)
     end
   end
 
-  attr_reader :models, :raw_response, :errors
+  attr_reader :models, :raw_response, :errors, :aggregations
 
   def initialize(raw_response)
     @raw_response    = raw_response
+    @aggregations    = raw_response["aggregations"]
```

Comment:
We've discussed this before, but I just wanted to double check: 

Should it be `Query::Response`'s responsibility to parse the raw `aggregations` response? Or maybe a shared aggregations parser that a consumer could use?

It would be additional work and would potentially require `Query::Response` to then also know about the aggregation field(s) (if we go that route), but it does feel a little weird that `distinct_values` will parse it and `where(aggs:)` will return the raw response.

Reply:
I wasn't convinced it was the `Query::Response`'s responsibility..I think we'll know better if we have a second use case for it

>but it does feel a little weird that distinct_values will parse it and where(aggs:) will return the raw response.

I actually disagree here. `distinct_values` is specify to returning the response from an aggregation whereas `where` is typically meant to return the matching models with the added benefit that now we can _also_ get aggregations.

This is especially true since we're only really supporting a single type of Aggregation right now. We would need to rework some of the aggregation assumptions here once we want to support more types of aggregations...not sure if we want to put in that work now or keep it YAGNI and do it when we have a need for a different aggregation.

Part of me wants to do the former since this is a public gem, but we're on critical path and I don't want to spend too much more time building out functionality that isn't going to be used

Reply:
ack, sounds reasonable then to leave it as is